vcl_templates/www.vcl.erb

```diff
@@ -296,6 +296,9 @@ sub vcl_deliver {
     add resp.http.Set-Cookie = "ABTest-Example=" req.http.GOVUK-ABTest-Example "; expires=" now + 1d;
   }
 
+  # Set the tls version cookie with the raw protocol version from fastly
+  add resp.http.Set-Cookie = "TLSversion=" tls.client.protocol "; max-age=0; secure";
```

Comment:
Also, do you know if there are any repercussions to adding `Set-Cookie` to every single request? I'm not aware of any problems that it would cause - it just seems a bit surprising.